src/Client.php

```diff
@@ -243,4 +243,43 @@ protected function getUserAgent()
             'php/'.PHP_MAJOR_VERSION.'.'.PHP_MINOR_VERSION
         ]);
     }
+
+    /**
+     * @param array|Entity $item
+     * @param array $map
+     * @return array
+     */
+    public function apiToFriendly($item, $map)
```

Comment:
better to call these snake to camel

Reply:
They aren't always snake to camel, but they are always api to friendly :)